### ISSUE TITLE
Arch FUSE install instructions

### ISSUE THREAD

**Issue by alschaapman** [AppImage/docs.appimage.org#72]
It is very important to note that the fuse2 package (provided by the extra repository) is NOT part of the Arch Linux base, nor is it a direct or indirect dependency of any packages in base. On my system, pactree -r fuse2 shows that fuse2 is depended on directly or indirectly by erofs-utils, exfat-utils, ntfs-3g, flatpak, python-llfuse, and libappimage, among a few others. I of course have many packages that optionally have an indirect dependency on fuse2, but it is always sensible to assume that Arch users will not install all the optional dependencies of all their software.

The bottom line is that, curiously, the Arch FUSE problems section of the FAQ states "On Arch Linux, FUSE should work already." This assumption is far LESS likely to be true on Arch, which installs almost absolutely nothing by default, than for basically any other distribution!

The FAQ should really just say
`sudo pacman -S fuse2`
